WcaOnRails/app/views/users/edit.html.erb

```diff
@@ -172,6 +172,12 @@
       <%= f.input :region, disabled: !editable_fields.include?(:region) %>
     <% end %>
 
+    <div class="form-group">
+      <% @user.teams.each do |team| %>
+        <%= link_to team.name, edit_team_path(team), class: "btn btn-info #{"disabled" unless current_user.can_edit_team?(team)}" %>
```

Comment:
Can you change these to plain links? When I saw the screenshots, I assumed clicking the buttons actually had a side effect. Maybe the links could say something like "is a member of TEAM", "Is not a member of TEAM".